File: FindWeather/FindWeatherMySolution/models/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FindWeatherMySolution.models
{
    public class Location
    {
        public string City { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public void ShowDetails()
        {
            Console.WriteLine("City: " + City);
            Console.WriteLine("Latitude: " + Latitude.ToString());
            Console.WriteLine("Longitude: " + Longitude.ToString());
        }
    }
}

File: FindWeather/FindWeatherSolution/com.findweather/models/Location.java
package com.findweather.models;

public class Location {
    private String city;
    private double latitude, longitude;

    public Location(String city, double latitude, double longitude) {
        this.city = city;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public String getCity() { return city; }
}
